packages/runtime/container-runtime/src/test/blobManager.spec.ts

```diff
@@ -452,7 +453,7 @@ describe("BlobManager", () => {
 
         const summaryData = validateSummary(runtime);
         assert.strictEqual(summaryData.ids.length, 1);
-        assert.strictEqual(summaryData.redirectTable.size, 2);
+        assert.strictEqual(summaryData.redirectTable.size, 4);
```

Comment:
Why is this 4 not 6?

Reply:
This one is interesting. This test changes the detached storage to DeDupStorage(). Basically, both detached and attached storage are different instances of DeDupStorage(). DeDupStorage uses a hash of the blob to generate a blobId which means it will generate the same ID for the same blob. Here are the blobIds generated in this test:
1. For the two blobs uploaded in detached mode - same id (localId). Let's call it blobId1.
2. For the 4 blobs uploaded in attached mode - 4 different localIds but the same storage id blobId1.

Now, we have 5 localIds and 1 storageId. However, the localId for the detached mode blobs are the same as storageId. So, essentially, there are 4 localIds which is what is added to the redirect table in summary.